app/views/api/v1/users/_info.json.jbuilder

```diff
@@ -0,0 +1,5 @@
+json.id       user.id
+json.email    user.email
+json.name     user.name
+json.uid      user.uid
```

Comment:
`uid` shouldn't be returned. It is used for authentication, it's part of the header response